terraform/outputs.tf

```diff
@@ -0,0 +1,25 @@
+output "app_name" {
+  value = juju_application.temporal_k8s.name
+}
+
+output "provides" {
+  value = {
+    metrics_endpoint  = "metrics-endpoint"
+    grafana_dashboard = "grafana-dashboard"
+  }
+}
+
+output "requires" {
+  value = {
+    admin                 = "admin"
+    db                    = "db"
+    frontend_certificates = "frontend-certificates"
+    ingress               = "ingress"
```

Comment:
Let's not include ingress for now as the traefik integration will be removed from this version.

Reply:
removed in a966a5e77731082712c23b0f5ad6daa52cb57ad8